### ISSUE TITLE
Modify the README.md and add a COD.md

### ISSUE THREAD

**Issue by Taity045** [100DaysOfCloud/100DaysOfCloudBackend#11]
#So i already created 2 pull requests i wasn't aware i should open an issue before doing that , @antoniolofiego  pointed that out after i already did. Basically what i did was modify the READ.md and i noticed that there was no code of conduct , so jotted something real quick that can be modified at any point.

**Comment by antoniolofiego** [comment]
I am not sure if a Code of conduct is needed in the codebase. We already have one on the website as well as in the discord channel. If anything, Contribution Rules might be beneficial.

**Comment by Taity045** [comment]
sure let me work on it to fit for just contribution rules

**Comment by antoniolofiego** [comment]
https://github.com/excalidraw/excalidraw/blob/master/CONTRIBUTING.md

This can be an idea, to be adjusted for the Issue -> PR workflow that we adapted.

**Comment by what-name** [comment]
I don't think we need a CoC in this repo. We do need however contributing guidelines and rules. Not sure what yet tho, open to suggestions or maybe find a really solid one and translate it over to here?

**Comment by Taity045** [comment]
I do think contribution guidelines are important , what if someone wants to contribute and they're not part of the discord ?
I do understand you can create an issue first but , i figure it's better to have an overall idea before you go all the way into creating an issue , i don't know though maybe it's just me.

**Comment by what-name** [comment]
Yeah, that's what I meant too haha, sorry if it came out the opposite way. Contribution guidelines ARE important - Code of conduct is what's not (for this repo).

My couple ideas of what should the guidelines contain (not comprehensive)
- A clear description of the what the PR and its changes entail
- Why are these changes important
- Make sure that you create tests where they are needed with the PyTest library. If you'd like to use another lanugage that is not python, please create an integration pipeline for it as well (see current one as example)
- Make sure to reference an existing issue or create a new issue before submitting your PR if there is none.

**Comment by what-name** [comment]
See the Frontend's PR template, it's great: https://github.com/100DaysOfCloud/100DaysOfCloudFrontend/pull/28#issue-463252904

**Comment by Taity045** [comment]
Thanks , Chris this looks like a good start, closing the issue now
